tutorials/optimization/2_converters_for_quadratic_programs.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Optimization problems in Qiskit's optimization module are represented with the `QuadraticProgram` class, which is generic and powerful representation for optimization problems. In general, optimization algorithms are defined for a certain formulation of a quadratic program and we need to convert our problem to the right type.\n",
+ "Optimization problems in Qiskit's optimization module are represented with the `QuadraticProgram` class, which is generic and powerful representation for optimization problems. In general, optimization algorithms are defined for a certain formulation of a quadratic program and we need to convert our problem to the right type.\n",
```

Comment:
Could we precede generic with "a","which is a generic and powerful representation"

Reply:
> 
> 
> Could we precede generic with "a","which is a generic and powerful representation"

@3yakuya I am also agreed with you in this regard. But, (it's my thinking) who have written the section has His/Her own way to express what he wanted to say. May be the expression can be only be made if there is no "a ". What do you say about it? Or we should treat it as what grammar demands as it is an open source??

Reply:
I have found lots of problems like this.. But, didn't touch them, thinking as the writer should have some freedom to express at his own way.. What do you think about it?? or we should treat it according to the grammer as it is open source??

Reply:
Good question, though I would think that we are not overriding author's freedom of expression, but rather trying to improve for general good :) I think the "a" makes it sound much more natural.